checker_test.go

```diff
@@ -0,0 +1,84 @@
+package tester
+
+import (
+	"testing"
+)
+
+func TestSegmentEqual(t *testing.T) {
+	testCases := []struct {
+		Name  string
+		A     map[string]interface{}
+		B     map[string]interface{}
+		Equal bool
+	}{
+		{
+			Name: "Ignores some fields",
```

Comment:
Could we have 2 test cases for `cleanMsg`? One where we delete fields in `context` (this test case here), and one for other fields where the values could be different? That way when the test fails we know immediately which part fails without any debugging.

Reply:
updated